tests/e2e/set-ffi-env-e2e

```diff
@@ -244,9 +244,6 @@ EOF
 
 install_autosd_repo() {
     # Install autosd repository
-    source /etc/os-release
-
-    releasever=${VERSION_ID}
```

Comment:
this remove seems incorrect as will break the line 253 (autosd.repo)

Reply:
No, it won't. the $releasever listed in autosd.repo is going to be resolved at runtime by dnf:
```console
 /usr/libexec/platform-python -c 'import dnf, json; db = dnf.dnf.Base(); print(json.dumps(db.conf.substitutions, indent=2))'
{
  "arch": "x86_64",
  "basearch": "x86_64",
  "releasever": "9"
}
```

Reply:
awesome, thanks sandro!